public/src/components/teacher/Results.tsx

```diff
@@ -209,7 +209,7 @@ export class Results extends React.Component<IResultsProps, IResultsState> {
         let selector: (string | JSX.Element | ICellElement)[] = [displayName, groupName];
         selector = selector.concat(student.labs.map(
             (e) => {
-                const setScore = e.assignment.getSkiptests() ? scoreFromReviews(e.submission?.reviews ?? []) : e.submission?.score ?? 0;
+                const setScore = e.assignment.getReviewers() > 0 ? scoreFromReviews(e.submission?.reviews ?? []) : e.submission?.score ?? 0;
```

Comment:
Maybe it would make more sense with a helper function on the `e` type here that returns the score.